http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/api/RuntimeRoutes.scala

```diff
@@ -36,7 +36,15 @@ class RuntimeRoutes(saturnIframeExtentionHostConfig: RefererConfig,
 ) {
   // See https://github.com/DataBiosphere/terra-ui/blob/ef88f396a61383ee08beb65a37af7cae9476cc20/src/libs/ajax.js#L1358
   private val allValidSaturnIframeExtensions =
-    saturnIframeExtentionHostConfig.validHosts.map(s => s"https://${s}/jupyter-iframe-extension.js")
+    saturnIframeExtensionHostConfig.validHosts
+      .filter(_ != "*")
+      .map(uri => if (uri.last == '/') uri.slice(0, uri.length - 1) else uri)
+      .map(s => s"https://${s}/jupyter-iframe-extension.js")
+
+  private def isValidSaturnIframeExtension(uri: String): Boolean =
+    !saturnIframeExtensionHostConfig.enabled ||
+      saturnIframeExtensionHostConfig.validHosts.contains("*") ||
+      allValidSaturnIframeExtensions(uri)
```

Comment:
These changes I think constitute a bugfix - the check for valid iframe extensions previously ignored wildcards and the refererConfig's `enabled` flag.